src/rust/engine/process_executor/src/main.rs

```diff
@@ -590,3 +596,50 @@ where
         })
         .collect()
 }
+
+#[cfg(test)]
```

Comment:
I didn't see any existing tests for cli arg parsing, let me know if I should move this somewhere else